qemu/tests/netkvm_rss_test.py

```diff
@@ -0,0 +1,48 @@
+from virttest import data_dir
+
+
+def run(test, params, env):
+    """
+    Test net adapter after set NetAdapterrss, this case will:
+
+    1) boot up guest
+    2) set NetAdapterrss
+    3) use speedtest trigger some network traffic
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environmen.
+    """
+    def execute_command(command, timeout=60, omit=False):
+        test.log.info("command is %s" % command)
+        status, output = session.cmd_status_output(command, timeout)
+        if status != 0 and omit:
+            test.fail("execute command fail: %s" % output)
+        return output
+
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_serial_login()
+
+    # Copy speedtest to guest
+    speedtest_host_path = data_dir.get_deps_dir("speedtest")
+    dst_path = params["dst_path"]
+    test.log.info("Copy Speedtest to guest.")
+    s, o = session.cmd_status_output("mkdir %s" % dst_path)
+    if s and "already exists" not in o:
```

Comment:
I think we should use `and`here. Since `s==1` means the command execution failed, if `already exists` exists in the output it means dir exists. So we can continue testing. If `already exists` is not exists in the output, it means we hit other errors,we should execute `test.error`.
If we use `or` here, if `s==0`, it means the command execution successful and the output should be empty,`“already exists" not in o` is true, in this case we should not execute `test.error`.

Reply:
@leidwang Right, I get it, thanks for the explaining.